strings/strings.json

```diff
@@ -4222,10 +4222,10 @@
   "wiki_faq_q_dark_energy": "What determines the amount of Dark Energy I receive?",
   "wiki_faq_a_dark_energy": "The amount of Mass and Exotic Mass in the hole, according to this formula: (LN(1 + (exotic mass * 40))) + (log2(mass - 7)/2.5)).",
   "wiki_basics_start": "Getting Started",
-  "wiki_basics_start_para1": "Evolve is a civilization building game, the goal is to develop your sepecies into the best it can be.",
+  "wiki_basics_start_para1": "Evolve is a civilization building game, the goal is to develop your species into the best it can be.",
   "wiki_basics_start_para2": "This guide will give an overview of the gameplay basics, if you wish to discover things for yourself then you should stop reading.",
   "wiki_basics_prehistoric": "Prehistoric Era",
-  "wiki_basics_prehistoric_para1": "Before you can start conquering the world you must first evolve into a sapient species.",
+  "wiki_basics_prehistoric_para1": "Before you can start conquering the world you must first evolve into a sentience species.",
```

Comment:
This doesn't really grammatically make sense, sentient would work here.

Reply:
Sorry, what did you mean? What did I write doesn't grammatically make sense, or what was written? 
I had changed that for consistency, since the word sapient did not appear anywhere else, but sentient did.
I should keep it, or revert it?